src/styles/_sicp.scss

```diff
@@ -353,3 +353,10 @@ $sicp-content-lr-padding: 6em;
     width: 70% !important;
   }
 }
+
+.bp4-omnibar {
+  @media only screen and (max-width: 768px) {
+    left: 1%;
+    width: 98%;
+  }
+}
```

Comment:
Please do not use Blueprint's CSS API (see #2722, https://github.com/source-academy/frontend/pull/2775#discussion_r1487232448).

Also, didn't we upgrade to Blueprint v5? Does this style make a difference?

Reply:
I have added a class to the search bar component and updated the style to reference the class.